### DISCUSSION TITLE
txt2al not erroring but also not generating the AL output for a CAL Delta of a standard codeunit

### DISCUSSION THREAD

**Discussion by lebragan** [microsoft/ALAppExtensions#26402] [Q&A]
Sorry guys for bringing a question about working with the txt2al tool, under the BC Spring 2019 release.

I've been finally cleared to do some work in creating AL extensions of a small number of objects from CAL to AL. From these, there is a CAL Delta of a codeunit (C408) that contains **changes to several standard functions**.

Anyway, the txt2al processes this codeunit - for testing it is the only CAL delta on the source path - no errors displayed, but no AL extension generated to the output path.

Would strongly appreciate your comments.

**Comment by JesperSchulz** [comment]
Sorry, this isn't the right repo to ask this question. I don't think you'll get any answers here. If my memory doesn't fail me, it was the compiler team which build the tool. Maybe try asking on https://github.com/microsoft/AL.

Or maybe try one of the public forums?

[Business Central Community](https://community.dynamics.com/forums/thread/?groupid=e78817ab-a926-4d31-96cc-aef040a4eb04)
[mibuso forum](https://forum.mibuso.com/)
[Viva Engage](https://www.yammer.com/dynamicsnavdev)

This GitHub repository handles issues with extending published apps, and extensibility in general, for the latest release of Business Central.
